torchtune/models/gemma/_component_builders.py

```diff
@@ -12,12 +12,14 @@
 from torchtune.modules import (
     MultiHeadAttention,
     FeedForward,
+    TiedLinear,
     FrozenNF4Linear,
     RotaryPositionalEmbeddings,
     TransformerSelfAttentionLayer,
 )
 from torchtune.models.gemma.rms_norm import GemmaRMSNorm
-from torchtune.models.gemma.transformer import GemmaTransformerDecoder
+from torchtune.models.gemma.gemma_norm_embedding import GemmaNormEmbeddings
```

Comment:
we put together the embeddings + norm that gemma requires, so that we can delete the GemmaTransformerDecoder